### PR TITLE
[New feature] Add support for Embeddings model from Amazon Bedrock & documentation

### PR DESCRIPTION
**Pull request by kandakji**
This pull request is to support a new embeddings model Amazon Bedrock expanding the capabilities of our langflow integration. The documentation is also included.

### CODE REVIEW

**Comment by ogabrielluiz** [comment]
Hey @kandakji 

Thanks for this PR.

Could you take a look at the other PR adding Amazon Bedrock? Their implementation is closer to what we are doing after we release CustomComponents.
#1053

**Comment by kandakji** [comment]
Sure @ogabrielluiz . I'll check it out and make the necessary changes accordingly.

Thanks

**Comment by kandakji** [comment]
@ogabrielluiz any reason why a custom component is needed for Bedrock Embedding ?

I was looking into an embedding implementation similar to when Cohere embedding was added.

**Comment by kandakji** [comment]
Hi @ogabrielluiz ,

Can you please let me know if we should go with a custom component or the default implementation like the other embedding models (Cohere, OpenAI)?

I have a branch ready with the custom component implementation if needed.

**Comment by ogabrielluiz** [comment]
Sorry for the delayed response, @kandakji. CustomComponents are the standard way now.

File: pyproject.toml [hunk 1]
```diff
@@ -93,6 +93,7 @@ markupsafe = "^2.1.3"
 boto3 = "^1.28.63"
 numexpr = "^2.8.6"
 qianfan = "0.0.5"
+boto3 = "^1.28.63"
```
Comment (ogabrielluiz) [inline pyproject.toml:96]: I think all you gotta do is remove this and this should be good to go. 

Thanks for the PR

**Review by ogabrielluiz — CHANGES_REQUESTED** [review]

File: pyproject.toml (hunk 1 above)
Comment (kandakji) [reply pyproject.toml]: resolved

**Pull request merged**